docs/install/Knative-custom-install.md

```diff
@@ -283,6 +283,15 @@ commands below.
    kubectl delete deploy knative-ingressgateway -n istio-system
    ```
 
+   You will also need to delete the following resource:
+   ```
+   kubectl delete statefulset/controller-manager -n knative-sources
+   ```
+   While the deletion of this resource during the upgrade process will
+   not prevent your users from successfully requesting updates to the Knative
+   Eventing resources, those changes will not be acted upon, or realized,
```

Comment:
It's actually the CRDs in eventing-sources that are affected:

- ContainerSource
- CronJobSource
- GitHubSource
- KubernetesEventSource
- AwsSqsSource
- GcpPubSubSource

Eventing CRDs that are not affected:
- Channel
- ClusterChannelProvisioner
- Subscription

Think it's worth listing the resources that are affected? It's a long list so it may just be noise.

Reply:
I think listing those out is probably too noisy.

Reply:
Yeah - listing them seems  like a lot and I'm not sure what the person would do with that info.